src/pages/RFOX/components/Stake/StakeInput.tsx

```diff
@@ -0,0 +1,138 @@
+  const translate = useTranslate()
+  const history = useHistory()
+  const asset = useAppSelector(state => selectAssetById(state, foxAssetId))
+  const [showWarning, setShowWarning] = useState(false)
+  const percentOptions = useMemo(() => [1], [])
+  const [cryptoAmount, setCryptoAmount] = useState('')
+  const [fiatAmount, setFiatAmount] = useState('')
+
+  const handleAccountIdChange = useCallback(() => {}, [])
+
+  const hasEnterValue = useMemo(() => !!fiatAmount || !!cryptoAmount, [cryptoAmount, fiatAmount])
+
+  const handleChange = useCallback((value: string, isFiat?: boolean) => {
+    if (isFiat) {
+      setFiatAmount(value)
+    } else {
+      setCryptoAmount(value)
+    }
+  }, [])
+
+  const handleWarning = useCallback(() => {
+    setShowWarning(true)
+  }, [])
+
+  const handleSubmit = useCallback(() => {
+    history.push(StakeRoutePaths.Confirm)
+  }, [history])
+
+  const assetSelectComponent = useMemo(() => {
+    return <TradeAssetSelect assetId={asset?.assetId} isReadOnly onlyConnectedChains={true} />
+  }, [asset?.assetId])
+
+  if (!asset) return null
+
+  return (
+    <SlideTransition>
+      <WarningAcknowledgement
+        message={translate('RFOX.stakeWarning')}
+        onAcknowledge={handleSubmit}
+        shouldShowWarningAcknowledgement={showWarning}
+        setShouldShowWarningAcknowledgement={setShowWarning}
+      >
+        <Stack>
+          {headerComponent}
+          <TradeAssetInput
+            assetId={asset?.assetId}
+            assetSymbol={asset?.symbol ?? ''}
+            assetIcon={asset?.icon ?? ''}
+            percentOptions={percentOptions}
+            onAccountIdChange={handleAccountIdChange}
+            formControlProps={formControlProps}
+            layout='inline'
+            label={translate('transactionRow.amount')}
+            labelPostFix={assetSelectComponent}
+            isSendMaxDisabled={false}
+            onChange={handleChange}
+            cryptoAmount={cryptoAmount}
+            fiatAmount={fiatAmount}
+          />
+          <FormDivider />
+          <AddressSelection />
+          <Collapse in={hasEnterValue}>
+            <StakeSummary assetId={asset.assetId} />
+            <CardFooter
+              borderTopWidth={1}
+              borderColor='border.subtle'
+              flexDir='column'
+              gap={4}
+              px={6}
+              py={4}
+              bg='background.surface.raised.accent'
+            >
+              <Row fontSize='sm' fontWeight='medium'>
+                <Row.Label>{translate('common.gasFee')}</Row.Label>
+                <Row.Value>
+                  <Amount.Fiat value='10' />
+                </Row.Value>
+              </Row>
+              <Row fontSize='sm' fontWeight='medium'>
+                <Row.Label>{translate('common.fees')}</Row.Label>
+                <Row.Value>
+                  <Amount.Fiat value='0.0' />
+                </Row.Value>
```

Comment:
Fine to keep it just in case, but as far as we're concerned at the moment, there are no explicit fees for depositing into the contract